docs/pages/jpsreport/jpsreport_method_A.md

```diff
@@ -0,0 +1,39 @@
+---
+title:  Method A
+keywords: analysis
+tags: [jpsreport, measurement]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: This method calculates the mean value of flow and density over time. 
+permalink: jpsreport_method_A.html
+last_updated: Dec 21, 2019
+---
+
+
+
+
```

Comment:
Why the image was removed from the detailed description of method A?

Reply:
Thanks! Added them now.